### ISSUE TITLE
Spyder fonts have changed and are now too blocky 

### ISSUE THREAD

**Issue by briancroxall** [spyder-ide/spyder#8132]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I just started Spyder and noticed that the fonts are all a lot blockier. The only major thing that I can think of that has changed in my system since the last time I used Spyder was that I installed Mac OS 10.14 (Mojave). 

### What steps reproduce the problem?

1.  Open Spyder and look at font. 

Sorry to be glib. In trying to troubleshoot this issue, I...
- restarted Spyder
- restarted my computer
- updated Spyder 
- updated Anaconda and all conda packages/dependencies
- reset Spyder's settings
- uninstalled and reinstalled Spyder within my conda environment

None of these 

### What is the expected output? What do you see instead?

What I expect is fonts that resemble the [images on the Spyder Documentation](https://docs.spyder-ide.org/editor.html).

Instead, the fonts are a little thicker and blockier.
![spyder python 3 6](https://user-images.githubusercontent.com/1005115/47387890-eac1a880-d6cd-11e8-84d9-4ca0e1cdde3a.png)

I might not have noticed this if I hadn't been using Spyder so regularly beforehand, but things look really unpleasant at the moment.

### Paste Traceback/Error Below (if applicable)

Not applicable

## Versions

* Spyder version: 3.3.1
* Python version: 3.6.6
* Qt version: 5.9.6
* PyQt version: 5.9.2
* Operating System name/version: Mac OS 10.14 (Mojave)

### Dependencies

```
IPython >=4.0     :  7.0.1 (OK)
cython >=0.21     :  0.29 (OK)
jedi >=0.9.0      :  0.13.1 (OK)
matplotlib >=2.0.0:  3.0.0 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
numpy >=1.7       :  1.15.3 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
psutil >=0.3      :  5.4.7 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pyflakes >=0.6.0  :  2.0.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
pylint >=0.25     :  2.1.1 (OK)
qtconsole >=4.2.0 :  4.4.2 (OK)
rope >=0.9.4      :  0.11.0 (OK)
sphinx >=0.6.6    :  1.8.1 (OK)
sympy >=0.7.3     :  1.3 (OK)
```

**Comment by ccordoba12** [comment]
What happens if you select a different font in

    Tools > Preferences > General > Appearance

?

**Comment by briancroxall** [comment]
When selecting a different font, the fonts change but they still seem to be rendered in a darker and slightly blockier version than before. It's most apparent on black text. A colleague came in and looked at my screen and could immediately see the difference.

**Comment by ccordoba12** [comment]
Ok, that means to me that this is not a Spyder problem but a Qt one (Qt is the graphical toolkit Spyder is based on).

We can't do anything about it, sorry. It should be fixed in the next Qt version (5.12).

**Comment by briancroxall** [comment]
Thanks, @ccordoba12.
